packages/ui/src/services/color-config/tests/color-functions.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { setHSLAColor } from '../color-functions'
+
+describe('ColorFunctions', () => {
+  describe('setHSLAColor', () => {
+    it('changeHEXhue', () => {
```

Comment:
Rename test names to human language. Like "Like change hue in hex color".

Reply:
Got it!